collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -906,6 +906,9 @@ protected void onActivityResult(int requestCode, int resultCode, final Intent in
                     getCurrentViewIfODKView().setBinaryData(bearing);
                 }
                 break;
+                case RequestCodes.FONT_SIZE_MATCHER:
+                    loadForm();
```

Comment:
I'm don't think we should be loading the form when returning here. This is quite intensive and also causes from to reload at the "hierarchy" view so the person filling the form out would lose their place. What we could do is call `refreshCurrentView` instead as that will just re-render the view.